core/src/test/scala/pureconfig/DurationConvertSuite.scala

```diff
@@ -82,4 +82,7 @@ class DurationConvertSuite extends FlatSpec with Matchers with EitherValues {
         ex.head shouldBe a[CannotConvert]
     }
   }
+  it should "fromS(fromD(Duration.Undefined)) == Duration.Undefined" in {
```

Comment:
Minor nitpick: can you put the spec title in english, just for consistency with the other specs in pureconfig? Some suggestions are `it should "handle correctly Duration.Undefined values"` or `it should convert correctly Duration.Undefined to and from a config`.